numpy/core/numeric.py

```diff
@@ -2173,6 +2173,8 @@ def allclose(a, b, rtol=1.e-5, atol=1.e-8, equal_nan=False):
     ``allclose(a, b)`` to evaluate to True.  The same is true for
     `equal` but not `array_equal`.
 
+    ``allclose`` is currently not defined for non-numeric data types.
```

Comment:
Drop the word "currently".  Even if we are planning to define this function for non-numeric types some time in the future, there is no need to hint at that in the current docstring.  Also, since it is just a reference to the name and not actual code, use single backticks around `allclose`.
```suggestion
    `allclose` is not defined for non-numeric data types.
```